profiler/src/profile_grouped_conv_bwd_weight.cpp

```diff
@@ -178,6 +179,21 @@ int profile_grouped_conv_bwd_weight(int argc, char* argv[])
             return profile(I2, NHWGC{}, GKYXC{}, NHWGK{}, BF16{}, F32{}, BF16{}, BF16{}, BF16{});
         }
     }
+    else if(num_dim_spatial == 2 && layout == ConvLayout::NGCHW_GKYXC_NGKHW)
+    {
+        if(data_type == ConvDataType::F32_F32_F32)
```

Comment:
I haven't seen that you've added F32 instances or have you?

Reply:
Removed.